### ISSUE TITLE
Backend - Persister is not working properly

### ISSUE THREAD

**Issue by rascaraficci** [dojot/dojot#259]
persister_1          | 2018-02-21T13:59:44.994445753Z Failed to bootstrap tenants's consumers.
persister_1          | 2018-02-21T13:59:44.994501767Z HTTPConnectionPool(host='data-broker', port=80): Max retries exceeded with url: /topic/dojot.tenancy?global=true (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f703b0cdc90>: Failed to establish a new connection: [Errno -2] Name or service not known',))
persister_1          | 2018-02-21T13:59:46.706102597Z Failed to bootstrap tenants's consumers.
persister_1          | 2018-02-21T13:59:46.706151483Z HTTPConnectionPool(host='data-broker', port=80): Max retries exceeded with url: /topic/dojot.tenancy?global=true (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f914fd2a650>: Failed to establish a new connection: [Errno 111] Connection refused',))
persister_1          | 2018-02-21T13:59:48.298199919Z Failed to bootstrap tenants's consumers.
persister_1          | 2018-02-21T13:59:48.298260629Z HTTPConnectionPool(host='data-broker', port=80): Max retries exceeded with url: /topic/dojot.tenancy?global=true (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb2c54d3650>: Failed to establish a new connection: [Errno 111] Connection refused',))
persister_1          | 2018-02-21T13:59:50.777090398Z will create consumer ['kafka:9092'] history b06308d6-4ecb-40fa-8ac0-69615fca900b
persister_1          | 2018-02-21T13:59:50.785466776Z Failed to bootstrap tenants's consumers.
persister_1          | 2018-02-21T13:59:50.785505486Z HTTPConnectionPool(host='auth', port=5000): Max retries exceeded with url: /admin/tenants (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fac87d1c190>: Failed to establish a new connection: [Errno -2] Name or service not known',))
persister_1          | 2018-02-21T13:59:54.243448764Z kafka consumer created b06308d6-4ecb-40fa-8ac0-69615fca900b - 3.46670103073

**Comment by mmagr** [comment]
Breaking down the logs:
```
persister_1 | 2018-02-21T13:59:44.994445753Z Failed to bootstrap tenants's consumers.
persister_1 | 2018-02-21T13:59:44.994501767Z HTTPConnectionPool(host='data-broker', port=80): Max retries exceeded with url: /topic/dojot.tenancy?global=true (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f703b0cdc90>: Failed to establish a new connection: [Errno -2] Name or service not known',))
```

This is actually expected behavior - it notes that `data-broker` is not up yet, thus making persister unable to ascertain what topics to consume from. It repeats thrice.

```
persister_1 | 2018-02-21T13:59:50.777090398Z will create consumer ['kafka:9092'] history b06308d6-4ecb-40fa-8ac0-69615fca900b
persister_1 | 2018-02-21T13:59:50.785466776Z Failed to bootstrap tenants's consumers.
persister_1 | 2018-02-21T13:59:50.785505486Z HTTPConnectionPool(host='auth', port=5000): Max retries exceeded with url: /admin/tenants (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fac87d1c190>: Failed to establish a new connection: [Errno -2] Name or service not known',))
persister_1 | 2018-02-21T13:59:54.243448764Z kafka consumer created b06308d6-4ecb-40fa-8ac0-69615fca900b - 3.46670103073
```

This is where the actual error is. On the fourth attempt to bring persister up, `data-broker` is already up, but `auth` isn't (auth depends on slower to boot services, such as kong). That makes persister unable to retrieve the list of pre-existing tenants. Do notice that the service has attempted to "die", but was actually left waiting for the already established long-running kafka task (tenant watcher) to resume (which doesn't, unless an error occurs).

This is fixed by dojot/history-ws#7

**Comment by mmagr** [comment]
This should be fixed in dojot-nightly/docker-compose@4441afe

@rascaraficci can you please verify?

**Comment by rascaraficci** [comment]
It's Ok.
